### DISCUSSION TITLE
Really slow response from strapi API

### DISCUSSION THREAD

**Discussion by AmauriMarcos** [strapi/strapi#7884] [Legacy Topics]
**System**

- Node.js version:  v12.16
- NPM version: 6.14.6
- Strapi version: 3.1.4
- Database: mongo atlas
- Operating system: windows 10

I'm making a simple blog which I deployed to netlify,  https://highlive.netlify.app/ . I would like to know why it takes so long  to load the content .  I have a collection type called Post and it has 10 fields which two of them is related to categories and tags.  I don't know if it helps but I deployed strapi on Heroku.  I would really appreciate if someone could help me to figure out why is this happening because I have just 13 posts on my Post's collection.

**Comment by laurenskling** [comment]
define "slow". Aren't you experiencing heroku putting your site to sleep every half hour?

**Reply by AmauriMarcos** [reply]
Probably is that ! It makes sense, thanks laurenskling, at least now I know what to do to avoid this.

**Comment by derrickmehaffy** [comment]
If deployed on Heroku, and the API hasn't been accessed in while (could be minutes or hours) Heroku kills the running Strapi server and on a new initial request it will "cold boot" the Strapi application (meaning it pulls from Git, builds the admin, then starts the server). The build process can take quite a bit of time.

If you need speed, I would suggest not using Heroku and as @laurenskling said, we need more information to provide more help.

I'm moving this over to a discussion as it's not really an issue with Strapi but more of deployment problems.

**Reply by AmauriMarcos** [reply]
Yes, you're right, it's not an issue with strapi. Well, thanks for the reply !
